contrib/repro_test.bzl

```diff
@@ -130,17 +130,46 @@ def _impl(ctx):
     img_tar = img_name + ".tar"
     img_digest = img_name + ".digest"
 
-    # Commands to build the image targets and copy the files required for image
-    # comparison to a known location in the container (being 'extract_path').
-    commands = [
-        "cd \$(cat /%s)" % proj_root.basename,
-        "bazel build " + " ".join(build_targets),
+    # Create directories for temporary storing of output_base of each of the
+    # Bazel executions inside a container.
+    # These directories will be mounted, with using their absolute paths
+    # in the containers.
+    # This is necessary as the Bazel executions inside each container can
+    # in turn bring up a docker container and attempt to mount contents
+    # from their respective output_bases.
+    bazel_out1 = ctx.actions.declare_directory("bazel_out1")
+    bazel_out2 = ctx.actions.declare_directory("bazel_out2")
+
+    # dummy action to declare directories as outputs.
+    ctx.actions.run_shell(
+        command = "exit 0",
```

Comment:
Why not do
`command = "mkdir -p %s %s" % (bazel_out1.path, bazel_out2.path)`
instead?

Reply:
because the directories get created automatically by Bazel (seems kind of a problem with documentation), if I try to recreate them with mkdir I get an error.